include/spdk/nvme.h

```diff
@@ -386,11 +386,11 @@ struct spdk_nvme_qpair;
  * Signature for the callback function invoked when a timeout is
  * detected on a request.
  */
-typedef void (*spdk_nvme_timeout_cb)(void *cb_arg,
-				     struct spdk_nvme_ctrlr *ctrlr,
+typedef void (*spdk_nvme_timeout_cb)(struct spdk_nvme_ctrlr *ctrlr,
 				     struct spdk_nvme_qpair *qpair,
+				     void *cb_arg,
```

Comment:
Ben, sorry, the abort_count name was a misnomer or typing error. The variable name is timeout_count and has been changed in the nvme.h.